lib/rules/react-svg-path-precision.js

```diff
@@ -0,0 +1,60 @@
+const t = require("@babel/types");
+
+module.exports = {
+    meta: {
+        docs: {
+            description: "Ensure that SVG paths don't use too many decimal places",
+            category: "react",
+            recommended: false,
+        },
+        fixable: "code",
+        schema: [
+            {
+                type: "object",
+                properties: {
+                    precision: {
+                        type: "number"
+                    }
+                },
+                additionalProperties: false
+            },
+        ],
+    },
+
+    create(context) {
+        let precision = 2;
+        for (const option of context.options) {
+            if (typeof option === "object") {
+                if (option.hasOwnProperty("precision")) {
+                    precision = option.precision;
+                }
+            }
+        }
+        const pattern = `(\\.${"[0-9]".repeat(precision)})[0-9]+`;
+        const regex = new RegExp(pattern, "g");
+
+        return {
+            
+            JSXAttribute(node) {
+                if (t.isJSXAttribute(node) && t.isJSXIdentifier(node.name, {name: "d"})) {
+                    if (t.isJSXOpeningElement(node.parent) && t.isJSXIdentifier(node.parent.name, {name: "path"})) {
+                        const d = node.value.value;
+
+                        if (regex.test(d)) {
+                            context.report({
+                                fix(fixer) {
+                                    const replacementText = d.replace(regex, (subtring, group) => group);
+        
+                                    return fixer.replaceText(node.value, replacementText);
```

Comment:
Based on my comments below I think you're going to have better luck doing something like: `parseFloat(num).toFixed(precision)`.

For example:

```
>> parseFloat("0.456").toFixed(2)
"0.46"
```